src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -83,9 +83,9 @@ internal IEnumerable<IKeyValueAdapter> Adapters
         internal IEnumerable<string> KeyPrefixes => _keyPrefixes;
 
         /// <summary>
-        /// An optional client that can be used to communicate with Azure App Configuration. If provided, the connection string property will be ignored.
+        /// An optional configuration client provider that can be used to provide clients to communicate with Azure App Configuration.
```

Comment:
This property is only used in our tests. Users have no way to set it. Can we add comments to clarify? BTW, the naming is not "provider" anymore.

Reply:
Done!